src/structures/GuildChannel.js

```diff
@@ -444,6 +444,18 @@ class GuildChannel extends Channel {
    * @param {ChannelResolvable} [options.parent=this.parent] The parent of the new channel
    * @param {string} [options.reason] Reason for cloning this channel
    * @returns {Promise<GuildChannel>}
+   * @example
+   * // Clone a channel
+   * channel.clone({
+   *  name: 'Some name',
```

Comment:
Didn't notice earlier: channels can't have spaces (we reject on this).

Reply:
Fixed